tests/test_file_writer_data_packet.py

```diff
@@ -280,10 +269,10 @@ def test_FileWriterProcess__process_next_data_packet__writes_tissue_data_for_two
 def test_FileWriterProcess__process_next_data_packet__writes_reference_data_to_active_subset_of_wells_if_the_timestamp_idx_starts_part_way_through_the_chunk__and_sets_timestamp_metadata_for_reference_sensor_since_this_is_first_piece_of_data(
     four_board_file_writer_process,
 ):
-    # Eli (12/9/20) a new version of black separated these all out onto separate lines...not sure how to de-duplicate it
-    # fmt: off
-    file_writer_process, board_queues, from_main_queue, _, _, file_dir = four_board_file_writer_process
-    # fmt: on
+    file_writer_process = four_board_file_writer_process["fw_process"]
```

Comment:
for future: if there's a super common configuration, then we could potentially consider creating a special key in the dictionary that contains a tuple of those 4 items, something like `file_writer_process, board_queues, from_main_queue, file_dir = four_board_file_writer_process['process, board queues, from main queue, file dir']`

or a very thin fixture that wraps `four_board_file_writer_process` but just yields a tuple of those specific 4 values.  haven't fully considered pros/cons of those approaches, but just food for thought / brainstorms